sdk/identity/identity/test/manual/src/index.tsx

```diff
@@ -4,205 +4,350 @@
 import * as React from "react";
 import * as ReactDOM from "react-dom";
 
-import { InteractiveBrowserCredential, BrowserLoginStyle } from "@azure/identity";
-import { KeyClient, KeyVaultKey } from "@azure/keyvault-keys";
-
-const getRandomKeyName = () => `key${Math.random()}`.replace(/\./g, "");
+import {
+  InteractiveBrowserCredential,
+  BrowserLoginStyle,
+  InteractiveBrowserAuthenticationFlow,
+} from "@azure/identity";
+import { ServiceBusClient } from "@azure/service-bus";
 
 interface ClientDetails {
-  tenantId: string,
-  clientId: string,
-  loginStyle: BrowserLoginStyle,
-  keyName: string,
+  tenantId: string;
+  clientId: string;
+  queueName: string;
+  loginStyle: BrowserLoginStyle;
+  flow: InteractiveBrowserAuthenticationFlow;
+  numberOfExecutions: number;
+  cacheCredential: boolean;
+  parallel: boolean;
+  serviceBusEndpoint: string;
+  output: string;
 }
 
 interface ClientDetailsEditorProps {
-  clientDetails: ClientDetails,
-  onSetClientDetails: React.Dispatch<React.SetStateAction<ClientDetails>>
+  clientDetails: ClientDetails;
+  onSetClientDetails: React.Dispatch<React.SetStateAction<ClientDetails>>;
 }
 
 function storeClientDetails(clientDetails: ClientDetails) {
-  localStorage.setItem('clientDetails', JSON.stringify(clientDetails));
+  localStorage.setItem("clientDetails", JSON.stringify(clientDetails));
 }
 
 function readClientDetails(): ClientDetails {
-  const detailsJson = localStorage.getItem('clientDetails')
+  const detailsJson = localStorage.getItem("clientDetails");
   if (detailsJson) {
-    const details = JSON.parse(detailsJson)
+    const details = JSON.parse(detailsJson);
     details.credential = undefined;
     return details;
   }
-
   return undefined;
 }
 
-function getCredential(clientDetails: ClientDetails): InteractiveBrowserCredential | undefined {
-  return clientDetails.tenantId.length > 0 && clientDetails.clientId.length > 0 ? new InteractiveBrowserCredential(clientDetails) : undefined;
+function Radio(options: {
```

Comment:
And a warning.
![image](https://user-images.githubusercontent.com/10452642/107593124-79ba9d80-6bc3-11eb-8ee1-bff9bffdbe1b.png)

Reply:
Let's skip this for now, it's a react thing.